### PR TITLE
standalone: use network of station if possible

### PR DESCRIPTION
**Pull request by Thykof**
when I use the wallet in standalone, I still want to use the network chosen by Massa Station.

This is a technical use case that could be useful.

Current behavior:
Wallet is standalone: use buildnet

This PR introduce this behavior:
Wallet always try to fetch Massa Station network endpoint, it case of failure (Massa Station isn't running), use the buildnet.

### CODE REVIEW

**Review by B-Naoufal — APPROVED** [review]

File: pkg/network/network.go [hunk 1]
```diff
@@ -2,18 +2,18 @@ package network
 
 import (
 	"encoding/json"
-	"fmt"
 	"io"
 	"net/http"
-	"os"
 
+	"github.com/labstack/gommon/log"
```
Comment (gregLibert) [inline pkg/network/network.go:8]: What about using massa-station log pkg instead ?

File: pkg/network/network.go [hunk 2]
```diff
@@ -24,25 +24,25 @@ type NetworkInfo struct {
...
 // retrieve network info from endpoint
 func GetNetworkInfo() (*NetworkInfo, error) {
-	if os.Getenv("STANDALONE") == "1" {
-		return &NetworkInfo{Network: defaultNetwork, URL: defaultNodeUrl}, nil
-	}
-
 	resp, err := http.Get(massaStationNodeEndpoint)
 	if err != nil {
-		return nil, fmt.Errorf("failed to make GET request: %v", err)
+		log.Warnf(fallbackMsg+"Failed to make GET request: %v", err)
```
Comment (gregLibert) [inline pkg/network/network.go:29]:
```suggestion
logFallback("GET massa station node endpoint", err)
```

With:
```golang
func logFallback(action string, err error) {
     log.Warnf("Fallback to default network: failed to %s: %v", msg, err)
 }

File: pkg/network/network.go (hunk 1 above)
Comment (Thykof) [reply pkg/network/network.go:8]: I would require to initlialize it, so chose a logs file, I'll do it later

Comment (Thykof) [reply pkg/network/network.go:8]: https://github.com/massalabs/station-massa-wallet/issues/538

File: pkg/network/network.go (hunk 2 above)
Comment (Thykof) [reply pkg/network/network.go:29]: I did the fix here https://github.com/massalabs/station-massa-wallet/pull/530/commits/688a2a5b0b1f1fa50fe07fbd5a711f65509ed9a2

**Comment by Thykof** [comment]
QA OK

**Pull request merged**